src/run.py

```diff
@@ -31,7 +31,118 @@
     print(f"Filling depressions in DEM '{inp}'...",end='',flush=True)
     wblib.fill(inp,out,flats)
     print('done.')
+
+ # Aspect tool
+elif toolname == 'aspect':
+    # get the parameters
+    try:
+        inp = kwargs['filled_DEM']
```

Comment:
I would go for just DEM, as the tool does also work with non-filled DEMs

Reply:
I agree!
Changed in revised commit.